### DISCUSSION TITLE
Latest beta live local video.

### DISCUSSION THREAD

**Discussion by steverae68** [samemory/homebridge-eufy-security-control#241] [General]
Thank you so much, this is working amazingly well. 

Thank you, thank you, thank you.....

**Comment by schliemann** [comment]
Agreed. What awesome work you have been doing @lenoxys. Just updated for the first time in 2 months. Amazing progress. Homebridge restarts after I exit the live view. Have you experienced this?

**Reply by njuart** [reply]
Hi, having the same behavior. After fresh restart, I can turn on/off camera with switch and get notifications, but as soon as I tap livestream I get immediate No response and HB restart. I also see this in logs

![image](https://user-images.githubusercontent.com/21228802/133961659-11d02cad-4f6a-43f0-a37f-6069853d7b0c.jpeg)

**Reply by steverae68** [reply]
Yeah. Top work - however, didn't realise at the time but I too, get the issue of homebridge restarting after live view from homekit.

Weirdly though not all the time, sorry was so excited about local stream, didn't notice that issue. 
I'll post logs here next time this happens

**Reply by mjtarr** [reply]
Can anyone provide guidance on how to get the beta installed using the HOOBs interface. It ain't obvious. Right now my 2k wired video doorbell does not show up in the Home app at all...

**Reply by steverae68** [reply]
The instructions should be in the wiki.

But click on the plugins list, select the eufy plugin, click on the spanner icon then select install previous versions...

Select the latest beta from the pop up list.

This works fine with homebridge native running on raspbian through the ui-x gui.

Can't say about hoobs though, not sure if hoobs is cut down in anyway....

**Comment by schliemann** [comment]
Interesting. The error I'm receiving is this:

![6BA19026-240A-4E20-9E9E-9912A47C6A09](https://user-images.githubusercontent.com/10818044/134040709-8e589433-086b-4fb9-83a7-16ce58d98011.png)

**Comment by mjtarr** [comment]
Thanks. I figured it out. In Hoobs, if one goes to the configuration page, there is a versions tab. If you go to versions you can then click on the beta and it installs. Just different UI.

But sill no video in HomeKit. In fact, although it recognizes the bridge, it doesn’t show the doorbell or anything associated with the bridge. Oh well.

Best, Mike

**Comment by steverae68** [comment]
Good stuff, glad you got it sorted.

The beta is a heck of an update to the standard version with the local storage access is great and makes the response of getting the stream much quicker.

**Comment by qthatswho** [comment]
I'm actually having a heck of a time getting any streams working with v1.1.1-beta.31 (just hangs, not a lot showing up even debug logging) - are there any special steps or config options that I need to set in order to experience what sounds like wonderfully fast local video streaming?

**Reply by qthatswho** [reply]
Ok, self-inflicted wound here. The cameras that were not responding were all set up as HomeKit. Once I "ignored" them during the wizard setup, all is well now. The HK cameras respond much quicker than the non, but all around things are working well. Awesome work!

**Reply by steverae68** [reply]
Glad you got it sorted, it truly is a work of genius what has been done so far, I'm not sure we'll ever got HKSV for the doorbells as they don't have an rtsp stream available (unless one can be created from from the plugin and re-treated) but the local video is a massive step forward.   I've donated to this project multiple times over the last year and a half and wat he'd it grow amazingly..   so very clever people involved...

Thank you all for your hard work @lenoxys @schliemann @samemory

**Comment by mjtarr** [comment]
I don’t get anything. It lets me add the bridge into HomeKit. But no devices show up there (other than the bridge). 

Hmmm.



“The only thing necessary for the triumph of evil is for good people to do nothing.”
__________________________________________________________________________
Kavčić-Moura Professor of Cognitive and Brain Science
Head, Department of Psychology
Carnegie Mellon Neuroscience Institute
Center for the Neural Basis of Cognition
Carnegie Mellon University
346c Baker Hall
5000 Forbes Avenue
Pittsburgh, PA  15213
412.268.4379

**Comment by mjtarr** [comment]
Ah. Fixed.

The guest account had gotten end-of-lifed when someone in the family accidentally took over the eufy master control…

So now I actually get live video!!!! Pretty cool.

I am getting this error in the log:

3/7/2022, 9:06:13 PMEufy Security Bridge 2ERROR(node:11847) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 livestream start listeners added to [Station]. Use emitter.setMaxListeners() to increase limit
3/7/2022, 9:06:13 PMEufy Security Bridge 2ERROR(Use `node --trace-warnings ...` to show where the warning was created)

No idea what that is…

-m
